test/unit/vl-map-layer-style.test.html

```diff
@@ -7,7 +7,6 @@
     <script src="../../../web-component-tester/browser.js"></script>
```

Comment:
Het lijkt erop (althans lokaal) dat bij het uitvoeren van de unit tests geen enkele test van deze file wordt uitgevoerd.

Reply:
en ook niet op bamboo: https://www.milieuinfo.be/bamboo/download/WEBCO-VLUIMAP-BD/build_logs/WEBCO-VLUIMAP-BD-373.log (als je zoekt op vl-map-layer-style.test, vind je niets terug).

Reply:
Klopt inderdaad, dit probleem werd reeds in een latere PR opgelost.